components/Feature/VulnerabilitiesGrid/index.js

```diff
@@ -166,42 +162,48 @@ const VulnerabilitiesGrid = ({ resources, onUpdate }) => {
                               value: label
                             })
                           }
+                          aria-label={arialabel}
                           aria-expanded={
                             textinputs
-                              ? selectedVulnerabilities
+                              ? grid.vulnerabilities[cbId]
+                                ? 'true'
+                                : 'false'
                               : undefined
                           }
                         />
-                        {textinputs &&
-                          grid.vulnerabilities[cbId] &&
-                          textinputs.map(({ label, type }) => {
-                            const inputId = `${cbId}-${labelToId(label)}-i`;
-                            return (
-                              <TextInput
-                                key={inputId}
-                                name={inputId}
-                                label={label}
-                                onChange={value =>
-                                  updateTextData({
-                                    key: inputId,
-                                    value,
-                                    label,
-                                    parentKey: cbId,
-                                    inputType: type,
-                                    gridType: 'vulnerabilities'
-                                  })
-                                }
-                              />
-                            );
-                          })}
+                        {textinputs && (
+                          <div aria-live="polite">
```

Comment:
Is that going to notify user when the textbook appears?

Reply:
yep